samples/Calling/package.json

```diff
@@ -10,8 +10,8 @@
     "start:app": "webpack-cli serve --mode=development --env development",
     "start:prod": "node dist/server.js",
     "test": "",
-    "prettier": "prettier --write --config ../../.prettierrc **/*.js **/*.ts",
-    "prettier:check": "prettier --check --config ../../.prettierrc **/*.js **/*.ts",
+    "prettier": "prettier --no-error-on-unmatched-pattern --write --config ../../.prettierrc --ignore-path=../../.prettierignore \"**/*.js\" \"**/*.jsx\" \"**/*.ts\" \"**/*.tsx\"",
```

Comment:
Is it possible to add this one into the config file? And I guess this is design for those small repos that doesn't have enough files, should we add this flag to all the repos? (I am a little worried about that we might introduce a wrong pattern in the future to match no files, but this flag just hides all of them)

Reply:
Some packages have only `.ts` files while some have `.tsx` as well. Some have `.js`, some don't.

I think it's better to just add all patterns for all packages right now. Otherwise whenever the first `.tsx` file is added, we'd have to remember to go update this script.

Reply:
Unfortunately, can't move this CLI arg to config.